stable/prometheus-aws-health-exporter/templates/deployment.yaml

```diff
@@ -19,6 +19,9 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "prometheus-aws-health-exporter.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        {{- if .Values.podLabels }}
+          {{ toYaml .Values.podLabels | nindent 8 }}
```

Comment:
Does the indentation before `{{ toYaml` screw up everything? 🤔 
I was thinking it should be an immediate newline without indentation space.

Reply:
nindent is almost identical to indent , but begins a new line. It may seem unnecessary, but look how much more readable it makes the file.

Reply:
Also CI tests would show an error if the indentation was wrong here 🙂